source/includes/_admin.md

```diff
@@ -74,12 +74,6 @@ adminApi = killbill.api.AdminApi()
 adminApi.trigger_invoice_generation_for_parked_accounts('test', api_key, api_secret)
 ```
 
```

Comment:
As per the current slate docs https://killbill.github.io/slate/#admin-trigger-an-invoice-generation-for-all-parked-accounts, this endpoint returns a list of invoices generated. So, is empty response appropriate here?

Reply:
Hi Reshma, yes I think that even the already existing No Content is also incorrect. It should show the list of invoices.

Reply:
Hi @reshmabidikar I was unable to park any account to test it. So for now I have pasted the invoice response from another endpoint that returns list of invoices for account.